model/src/service/tasks.py

```diff
@@ -128,30 +103,22 @@ def make_request(task_id, chain_no, address, start_time, request_body, task, req
        forward_headers.update({'Content-type' : 'application/json'})
 #       res = requests.post(dst, data=json.dumps(d, indent=4, default=str), headers=forward_headers)
 
-
-
-def save_task(json_data, address, headers):
-    with Connection(redis):
-        start_time = datetime.datetime.utcnow()
-        task_id = str(uuid.uuid4())
-        chain_no = json_data["chain_no"]
-        request_type = json_data["request_type"]
-        request_to_next_service = str(int(request_type) + 1)
-        request_task_type = list(gen_dict_extract("request_task_type", json_data))[0]
-        if int(request_type) > len(request_task_type):
-            task = random.choice(["cpu", "memory", "sleep"])
-        else:
-            task = list(gen_dict_extract(request_type, request_task_type))[0]
-        if task in ["cpu", "memory", "sleep"]:
-            q = Queue()
-        else:
-            q = Queue('high')
-#        result = make_request(task_id, chain_no, address, start_time, json_data, task, request_to_next_service)
-        enqueue_task = q.enqueue_call(func=make_request, args=(task_id, chain_no, address, start_time, json_data, task, request_to_next_service, headers), job_id=task_id, result_ttl=5000)
-        response_object = {
-            "status": "Task created",
-            "data": {
-                "task_id": task_id
-            }
+def execute_task(json_data, address, headers):
+    start_time = datetime.datetime.utcnow()
+    task_id = str(uuid.uuid4())
+    chain_no = json_data["chain_no"]
```

Comment:
We don't have any chain_no in the json, we have just the address of next service and so on. Something like a simple application like OnlineBoutique(known as Hipstershop)

Reply:
Right... this will also be fixed with issue #28.